pegasus/sites.v3/code.org/views/alternative_classrooms_professional_learning_banner.erb

```diff
@@ -0,0 +1 @@
+<%= view :professional_learning_apply_banner, :link_suffix => "middle-high" %>
```

Comment:
This view is only used on the alternative classrooms page. Ideally I wouldn't create a separate view just for this, but because of how partials work (see "Advanced Partials" section here https://code.org/editing/concepts), I have to create a separate view to include a param in the view.